vlad.metelap/image.rb

```diff
@@ -0,0 +1,21 @@
+require 'faraday'
+require "open-uri"
```

Comment:
You don't need `open-uri` here. Please fix other rubocop issue so I can review the code

Reply:
corrected done.